Makefile

```diff
@@ -1,6 +1,5 @@
 MKFILE_PATH 			:= $(abspath $(lastword $(MAKEFILE_LIST)))
 WORKING_DIR 			:= $(dir $(MKFILE_PATH))
-NUM_PROC                := $( $(command -v nproc &> /dev/null) && echo $(nproc) || echo $(sysctl -n hw.physicalcpu) )
```

Comment:
This is for compatibility of MacOS. Why remove this?

Reply:
It doesn't always work, instead, we leverage `nproc` in `coreutils` package on Mac.

Reply:
OK~